travis.sh

```diff
@@ -75,6 +75,16 @@ do
     command -v dub >/dev/null 2>&1 || { echo >&2 "DUB hasn't been installed."; exit 1; }
     deactivate
 
+    # Check man pages do not destroy system man pages availability
+    if man ls >/dev/null 2>&1; then man_ls_exit_status=0; else man_ls_exit_status=$?; fi
```

Comment:
This is a bit of an overkill. `set -e` is used in the script, so you don't need the if else at all. The script will fail of the command fails and you can use `! ..` for negation.

Reply:
Point of the matter is that `man dmd` before setting up MANPATH thru `activate` should cause the exit code to be `16` and I didn't know any better way to check that. So, the two first checks are just to pick up the current exit codes before `activate` to compare with after `deactivate`. The middle two checks are 1) `dmd`'s man page available, 2) we haven't lost the system man pages (thus unchanged from the first test).

Another thing, I didn't see any tests in travis for using the fish-shell. Should I introduce `man` page testing for that shell too (is it available?).

BTW, LDC means ...?